### ISSUE TITLE
Is the project maintained or discontinued?

### ISSUE THREAD

**Issue by esseti** [cachethq/cachet#4326]
Hi,
wondering if the project is maintained or it has been discontinued.
any info?

i see very old commits and little reply to issues.

**Comment by SpeedyCraftah** [comment]
Appears mostly dead. Sad since this looks like a great alternative to the paid status pages - could of course make use of more up-to-date technologies but still works.

**Comment by AGrant24** [comment]
I think it has been acquired by another company

**Comment by mbh-dev** [comment]
You're best using Uptime Kuma (https://github.com/louislam/uptime-kuma) Unfortunately this project is effectively dead. Shame as it was a great alternative. Things like subscriptions for notifications of incidents and maintenance, simply don't work. Maybe one day this will be revived.

**Comment by SpeedyCraftah** [comment]
Though I'm pretty sure you all can agree on the fact this services stack is rather outdated and old. When I was first setting it up, knowing little about PHP I ran into a lot of issues and it took me a while to get it running - especially with the caching servers.

**Comment by TheDelta** [comment]
Don't really agree on outdated stack or having issues with it.
Just a lack of documentation and further updating of this great project.
Sure it could be also a fancy super duper first class stack, but as long as it works.

Always sad to see that such os projects just kept idle and no response from the maintainers :(

**Comment by nixmomo** [comment]
How can you say it is not outdated? The last release was in 2019.... cachet was a realy great stack but its outdated and from my point of view, mostly dead.

**Comment by TheDelta** [comment]
> How can you say it is not outdated? The last release was in 2019.... cachet was a realy great stack but its outdated and from my point of view, mostly dead.

I was referring to the stack (php / laravel) - totally agree that it's outdated in terms of updates / bugfixes and features.
It still works with some quirks but it needs some love by some php developers (and sadly I don't have time).
I haven't checked the forks yet, eventually some(one) are keeping it alive already?

**Comment by j-a-pope** [comment]
@TheDelta I was also wondering if there is an active fork out there somewhere. Please post back in here if you find one.

**Comment by ouraios** [comment]
@jbrooksuk Can you see if you have time to transmit owning of this amazing project to someone who have enough time to maintain it ?

**Comment by awjudd** [comment]
> @jbrooksuk Can you see if you have time to transmit owning of this amazing project to someone who have enough time to maintain it ?

100% agree - I would love to help maintain this and start introducing some of the stuff from newer Laravel versions.

**Comment by j-a-pope** [comment]
I'm seriously considering creating and maintaining a fork of Cachet, because I think it's a great piece of software, feel it's sad that it has been largely abandoned, and want to use it in production. 

But, I am also very aware that I have a lot of other commitments and can not donate anywhere near as much time as it would take for someone to be the sole maintainer. 

If there are a couple of other people who would be seriously interested, willing, and have the ability to donate a reasonable amount of time to the project, then I would like to create a new fork which is owned and maintained by a 'steering committee'.

There are two purposes to this, one it means that it is not such a time drain for any one person, and, probably more importantly, it ensures that the project does not die or stagnate again at some future point because one maintainer no longer wants to or is able to take care of it. 

This would also include forking (or creating from scratch) and maintaining one of the extensions to monitor domains/servers and automatically update the status page with any issues.

Is there anyone else here who would want to, and be able to, join such a project?

**Comment by awjudd** [comment]
@John-S4 I'd be interested

**Comment by j-a-pope** [comment]
Ok, I'm pretty committed to doing it, so will work through some ideas and details in my mind and then see where we can take it. 

@awjudd I'll get in touch with you, probably tomorrow, if that's good.

**Comment by SpeedyCraftah** [comment]
Wish I had more experience with PHP/Laravel otherwise I'd devote some effort as a maintainer. Wish you the best of luck guys!

**Comment by j-a-pope** [comment]
@SpeedyCraftah There are going to be a lot of ways to contribute that don't involve a lot of coding - I might reach out to you as well if that is ok.

**Comment by mbh-dev** [comment]
Happy to chuck over some free hosting if that'd help at all 😊

**Comment by TheDelta** [comment]
I had no time to look over the forks yet, but I saw a lot of changes in some, including bug fixes etc.
So it would be vise to check and merge some into the "new" project.
If I find the time, I'll contribute some features / fixes out of personal interest myself:
- (better) time zone handling and display
- update support of incidents

**Comment by SpeedyCraftah** [comment]
> @SpeedyCraftah There are going to be a lot of ways to contribute that don't involve a lot of coding - I might reach out to you as well if that is ok.

Go ahead! Besides I can always learn PHP/Laravel, right?

**Comment by j-a-pope** [comment]
> I had no time to look over the forks yet, but I saw a lot of changes in some, including bug fixes etc.
> So it would be vise to check and merge some into the "new" project.

Yep, I noticed that there are a few more recent commits in some of the forks, but with 1500 forks to look through, and probably all conflicting with each other, I'm not sure how practical that is going to be. 

Did you notice any forks in particular with many improvements?

**Comment by TheDelta** [comment]
> > I had no time to look over the forks yet, but I saw a lot of changes in some, including bug fixes etc.
> > So it would be vise to check and merge some into the "new" project.
> 
> Yep, I noticed that there are a few more recent commits in some of the forks, but with 1500 forks to look through, and probably all conflicting with each other, I'm not sure how practical that is going to be.
> 
> Did you notice any forks in particular with many improvements?

Nope, I would just look at the 10 recent forks and review the current PR's (not by dependabot: https://github.com/search?l=&q=is%3Aopen+is%3Apr+-author%3Aapp%2Fdependabot+-author%3Aapp%2Fdependabot-preview+repo%3ACachetHQ%2FCachet&type=issues)

**Comment by j-a-pope** [comment]
Just as a quick update - I am still actively planning to pursue a new forked project as spoken about above and have started putting things together.

**Comment by awjudd** [comment]
@John-S4 hit me up and we can discuss more I had a bunch of plans for it back in the day.

**Comment by ruiztulio** [comment]
Hi @John-S4 !

I'm one of the [Spanish translators](https://github.com/CachetHQ/Cachet/issues/3088#issuecomment-397950937). If you fork it I'm willing to continue helping with the translations. 

As for PHP the last time I did something Symfony was about to release 2.0 and Laravel didn't even existed yet (yeah, long time ago XD ), but could give you a hand with doc, setup manual or stuff like that.

(I still have it in production and using it daily)

Regards

**Comment by awjudd** [comment]
(I'm a software developer whose been using Laravel for ages, and access to hosting for a bunch of different cachet-related work :))

**Comment by jbrooksuk** [comment]
Hey everyone 👋🏻 

Sorry for not getting back to anyone on this matter.

Basically, I sold Cachet to apilayer a few years back. The idea was that they would continue development and use it to build out an ecosystem. At the time of the sale, my brother passed away and my first daughter was born. I was in a strange place and so didn't continue my contracting work on Cachet. apilayer was understanding of my situation and honestly, my communication was not very good.

Since then, apilayer haven't touched Cachet. Over the last few weeks I've been reaching out to the contacts I have there to see what the situation is/was and whether I could take on the project once more.

I've not heard from them across any of the channels I've used:

- Email
- SMS
- Twitter DMs
- LinkedIn Messages

Before posting any updates, I wanted to make sure I'd tried to get an update from their end, but it seems that's not possible.

Since I no longer own the project or domain, I don't really know how to move forward. Perhaps, I can fork the project and get things moving from there. If we were to do this, then I would want to build a small team of developers who could help implement things with me. There is a lot of work to be done - mostly, because I worked against Laravel's conventions in some places. Honestly, I think we'd be better starting with the existing migrations and building from that.

So that's the situation. I'm sorry for the lack of communication from everyone who has messaged. For a long time, Cachet was a reminder of a whirlwind time in my life. However, I love this project and I am where I am now because of it ❤️

**Comment by j-a-pope** [comment]
@jbrooksuk 

Hi James, 

Good to hear from you, hear that life is going well, and to know that you still care about Cachet. 

I think that in the case that you are no longer the owner of the project name, domain, etc. then a fork is the simple and easy way to move forward. 

I am actually in the process of organising a fork myself (as in the last handful of posts above), and had picked a name, bought a domain, etc. 

If you are planning on forking and reinvigorating the project yourself then perhaps I will put that plan on hold right now, and would be very willing to be part of a team working on a new fork. 

If not then I will push forward with my current plan. 

In either case, I would like to see any new fork owned/maintained by more than one person, something like a 'steering committee', which would help in the case that one person was not available for whatever reason for an extended period of time in the future.

**Comment by Napsty** [comment]
Just to throw in some words into the discussion here: I think this "issue" is a wonderful thread of what open source is supposed to be. It is wonderful to see how multiple people, whether dev or non-dev, plan something together and there's no ego playing around "i did it first" etc. Will be interesting to see where this leads to (looks like this will be a good team!) and will be happy to use the result. Keep it up! :heart:

**Comment by jbrooksuk** [comment]
A quick update, I've finally spoken to Paul from apilayer (now acquired by Idera) and we now have ongoing communication about this issue.

They're currently busy with a big project and have asked if we can continue our discussion in the fall. This will take us to the end of September, the beginning of October before we can have a more meaningful and complete discussion.

**Comment by serjaum** [comment]
Good to know! following this thread :D

**Comment by SpeedyCraftah** [comment]
> Hey everyone 👋🏻
> 
> Sorry for not getting back to anyone on this matter.
> 
> Basically, I sold Cachet to apilayer a few years back. The idea was that they would continue development and use it to build out an ecosystem. At the time of the sale, my brother passed away and my first daughter was born. I was in a strange place and so didn't continue my contracting work on Cachet. apilayer was understanding of my situation and honestly, my communication was not very good.
> 
> Since then, apilayer haven't touched Cachet. Over the last few weeks I've been reaching out to the contacts I have there to see what the situation is/was and whether I could take on the project once more.
> 
> I've not heard from them across any of the channels I've used:
> 
> * Email
> * SMS
> * Twitter DMs
> * LinkedIn Messages
> 
> Before posting any updates, I wanted to make sure I'd tried to get an update from their end, but it seems that's not possible.
> 
> Since I no longer own the project or domain, I don't really know how to move forward. Perhaps, I can fork the project and get things moving from there. If we were to do this, then I would want to build a small team of developers who could help implement things with me. There is a lot of work to be done - mostly, because I worked against Laravel's conventions in some places. Honestly, I think we'd be better starting with the existing migrations and building from that.
> 
> So that's the situation. I'm sorry for the lack of communication from everyone who has messaged. For a long time, Cachet was a reminder of a whirlwind time in my life. However, I love this project and I am where I am now because of it heart

Completely fair - everyone needs to step back every once a while and focus your time on your life and job. There's not much you could have really done in this situation except we thank you for starting such a great project, I'm sure many SM enterprises appreciate the open source status pages without having to pay companies for pre-existing status page software exorbitant amounts of money  (I'm sure everyone knows who they are). It's a great thing open source exists so anyone can just kind of fork the project and carry on with the development as if nothing had happened.

**Comment by TheDelta** [comment]
I find it a bit concerning that this project has been abandoned and now the next talk round will start in October (truly sad that there is no way to find a few hours or even write back and forth)
This project truly needs some fixes asap, like PHP update which breaks cachet atm. :(

I mean holy cow, there are PRs from 2016! - time flies by for sure.

**Comment by jbrooksuk** [comment]
@SpeedyCraftah thank you, 🙏🏻 

@TheDelta I totally appreciate that. Although conversations might not take place till then, I'm still giving this thought and my time - hopefully something will come of this sooner.

**Comment by ouraios** [comment]
@jbrooksuk Even if we have to wait until end of Septembre to know what the next step is for Cachet, is it stopping us from cleaning the issues and PRs ? If not then maybe you could add @John-S4 to CachetHQ organization so he can starts helping with Cachet ?

**Comment by llann** [comment]
Waiting for some news. This one looks kind of maintained with a release 2.5.1 end of August 2021 and a branch 2.6 and has 100+ stars : https://github.com/fiveai/Cachet

**Comment by j-a-pope** [comment]
Hi all, 

I'm actually mostly away from my screen and hiking in the mountains for the last few days and the next week, so not doing much until then but I am following this, and do still want to be a part of moving Cachet forwards again in whatever form that might best take.

**Comment by awjudd** [comment]
I can help with PRs and stuff too if desired.

**Comment by Clevero** [comment]
@llann https://github.com/fiveai/Cachet you mentioned should be seen as dead. Quote:

> @jeroendev-one I've recently left FiveAI and I currently don't have any time to work on Cachet unfortunately, so it's not maintained by me anymore, sorry.

https://github.com/fiveai/Cachet/issues/80

**Comment by vedranmiletic** [comment]
@John-S4 any updates on the fork situation? I'm interested in helping with coding on a non-regular basis starting October.

**Comment by j-a-pope** [comment]
Hi @vedranmiletic 

I'm still very interested and as I said before have put some thoughts and plans together for a fork. 

Since @jbrooksuk posted in here that he is planning to revive or fork Cachet himself, I had put those plans on hold for the moment. That is mostly because I think that there is no real purpose in creating two separate active versions and it will just dilute and split efforts and involvement. 

Since there does not seem to be any immediate action here though, as James explained, I am not sure what the best decision and direction is, and if I should not start a new fork anyway.

**Comment by vedranmiletic** [comment]
@John-S4 sounds good. Let's see what happens over the next moth.

**Comment by j-a-pope** [comment]
Hi @jbrooksuk 

Is there any update on either your own thoughts/plans and/or discussions with apilayer?

**Comment by InFerYes** [comment]
@jbrooksuk @John-S4 is there a channel for direct communication? I would like to explore if I can contribute.

You mentioned the purchase of a domain and having a new name, is there a holdup to move forward?

**Comment by j-a-pope** [comment]
@InFerYes

As I wrote previously, I was all ready to go with a new fork of the project, had some thoughts, a name, domain, etc. 

Then James appeared here and said that he had a plan to revive the project himself, and I thought that rather than creating a fork and dividing attention and dev time it would make sense to hold off for a while and see what his plans were and what would happen to this existing project. 

However, we are now well into October and there seems to be little happening, so I am once again considering the options and thinking that maybe I will go ahead with a fork anyway.

**Comment by InFerYes** [comment]
@John-S4 I suggest you move forward and, if necessary, merge back later. In the meanwhile progress can be made.

**Comment by j-a-pope** [comment]
@InFerYes Yes, that is the way my thinking is leaning. I'm super busy with other things this week, so will wait until the end of the week and see if there is any update from @jbrooksuk and if not then I will go for it.

**Comment by ruiztulio** [comment]
I agree on this with @InFerYes, before the initiative loose momentum. I talk by myself here, was looking for an alternative and completely forgot about this issue until the recent messages.

If you plan to do something in the near future count me in.

**Comment by j-a-pope** [comment]
> If you plan to do something in the near future count me in.

I do. Hopefully I will find a bit of free time tomorrow and get it started.

**Comment by j-a-pope** [comment]
I'm assuming that since it has been discussed in this Issue it is ok to post here that I have created a new organization and will be working on a forked version of Cachet. 

If @jbrooksuk revives Cachet in the near future and if we all share the same vision and direction for where it should go then there is always the possibility of merging it back. If not then it will continue in its own direction and we will see where we end up.

I would like to see any new fork owned/maintained by more than one person, something like a 'steering committee', to help prevent possible future issues when one person is unable or does not want to continue with the project. I'm actively looking for people who would like to be a part of that, as well as contributors. 

@awjudd @TheDelta @SpeedyCraftah @ruiztulio @vedranmiletic @InFerYes and anyone I missed - If any of you are still interested then let's connect and make some plans. 

At this moment, there is nothing new happening. The repo is exactly as it cloned from Cachet. It's late where I am but I will be editing the readme tonight. Tomorrow I will try and make a start on triaging over some of the issues and PRs from here. 

You can find it over at https://github.com/Okazanta/Okazanta-core

Oh, and if you think the name is strange, don't worry, there is some logic behind it.

**Comment by mbh-dev** [comment]
Would like to help where I can and be involved in the new fork, let me know what you need doing 🙂

**Comment by j-a-pope** [comment]
> Would like to help where I can and be involved in the new fork, let me know what you need doing 🙂

Oh, a lot ! 😁

I will be opening some issues on the new fork, creating a starting roadmap, and create some discussion channels (possibly Discord), so that is the place to check.

**Comment by ruiztulio** [comment]
@John-S4 Yes, I'm in!

**Comment by arag00rn** [comment]
@John-S4 I'm in! Can't wait to see u the next dashboard :)

**Comment by j-a-pope** [comment]
@ruiztulio @arag00rn @mbh-dev 

There are already some thoughts and questions that it would be good to come up with answers to posted in the [Issues](https://github.com/Okazanta/Okazanta-core/issues) at the fork. Feel free to jump in with thoughts so that we can get things moving.

**Comment by some0ne2** [comment]
@jbrooksuk is there any update you can provide?

**Comment by MrBigbee** [comment]
Okazanta is also dead :
https://github.com/Okazanta/okazanta-legacy/issues/43

@jbrooksuk : Do you have some news on your side about Chachet ?

**Comment by jbrooksuk** [comment]
Hey 👋 

Please check out https://github.com/CachetHQ/Cachet/discussions/4342

TLDR; Cachet is back! 🎉
